### PR TITLE
[Impeller] removed vulkan unit test csv

### PR DESCRIPTION
**Pull request by gaaclarke**
fixes https://github.com/flutter/flutter/issues/113961

There were just 2 outstanding tests that were failing.

## Pre-launch Checklist

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I read and followed the [Flutter Style Guide] and the [C++, Objective-C, Java style guides].
- [x] I listed at least one issue that this PR fixes in the description above.
- [ ] I added new tests to check the change I am making or feature I am adding, or Hixie said the PR is test-exempt. See [testing the engine] for instructions on writing and running engine tests.
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] I signed the [CLA].
- [x] All existing and new tests are passing.

If you need help, consider asking for advice on the #hackers-new channel on [Discord].

[Contributor Guide]: https://github.com/flutter/flutter/wiki/Tree-hygiene#overview
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[C++, Objective-C, Java style guides]: https://github.com/flutter/engine/blob/main/CONTRIBUTING.md#style
[testing the engine]: https://github.com/flutter/flutter/wiki/Testing-the-engine
[CLA]: https://cla.developers.google.com/
[flutter/tests]: https://github.com/flutter/tests
[breaking change policy]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes
[Discord]: https://github.com/flutter/flutter/wiki/Chat

### CODE REVIEW

File: testing/run_tests.py [hunk 1]
```diff
@@ -1209,20 +1196,17 @@ def main():
...
-      vulkan_gtest_filter = parse_impeller_vulkan_filter()
-      gtest_flags = shuffle_flags
-      gtest_flags.append(vulkan_gtest_filter)
       run_engine_executable(
           build_dir,
           'impeller_unittests',
           engine_filter,
-          gtest_flags,
+          shuffle_flags + [
+              '--gtest_filter=-*/OpenGLES:Play/TypographerTest.MaybeHasOverlapping/Vulkan:Play/TypographerTest.GlyphAtlasWithLotsOfdUniqueGlyphSize/Vulkan',
```
Comment (gaaclarke) [inline testing/run_tests.py:1208]: I'll create follow up issues for these 2 tests.

Comment (zanderso) [reply testing/run_tests.py:1208]: SGTM. Could you make a TODO here that links to the issue?

**Review by zanderso — APPROVED** [review]
lgtm w/ TODO

File: testing/run_tests.py (hunk 1 above)
Comment (gaaclarke) [reply testing/run_tests.py:1210]: Done.

**Pull request merged**